docs/notebook_tutorial/tutorial_eliza.ipynb

```diff
@@ -0,0 +1,169 @@
+    "    \"\"\"\n",
+    "    Start a remote service for ElizaProcessor\n",
+    "    \"\"\"\n",
+    "    pipeline = Pipeline[DataPack]()\n",
+    "    pipeline.set_reader(RawDataDeserializeReader())\n",
+    "    pipeline.add(ElizaProcessor())\n",
+    "    pipeline.serve(input_format=input_format, service_name=service_name)\n",
+    "\n",
+    "\n",
+    "if __name__ == \"__main__\":\n",
+    "    Thread(target=start_eliza_service).start()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualize the chatbot in `stave`\n",
+    "\n",
+    "We will need `stave` to open a chatbot window for us to test the pipeline service that we just started in the previous step. `stave` is a fast, lightweight, extensible web-based text annotation and visualization tool designed to support a wide range of data types and NLP tasks. For more details, refer to https://github.com/asyml/stave.\n",
+    "\n",
+    "Run the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!stave -s start -o -l -n 8889"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we should see a browser window popped out which directs to a login page (http://localhost:8889/login) of `stave`. You can log in with default username (`admin`) and password (`admin`).\n",
+    "\n",
+    "After successfully logging into `stave`, we can navigate to the `All Projects` page. Click the `VIEW PROJECT` button under `Eliza` project. Then click `eliza.json` on the left side to enter the dialogue page where you can enter queries and get responses from Eliza chatbot.\n",
+    "\n",
+    "## Code Explained\n",
+    "### Overview\n",
+    "The Eliza example showcases `forte`'s ability to expose a pipeline to be served as a remote service that can be called from another pipeline using [`RemoteProcessor`](https://github.com/asyml/forte/blob/master/forte/processors/misc/remote_processor.py). This allows users to port their local pipeline to a remote endpoint that can be accessed and shared by other users to call its functionality. We will use the Eliza example to show how we can achieve this.\n",
+    "\n",
+    "### Start a Pipeline Service\n",
+    "In the code snippet above we build a simple pipeline and start it as a service for Eliza chatbot:\n",
+    "```python\n",
+    "pipeline = Pipeline[DataPack]()\n",
+    "pipeline.set_reader(RawDataDeserializeReader())\n",
+    "pipeline.add(ElizaProcessor())\n",
+    "pipeline.serve()\n",
+    "```\n",
+    "Here we set [`RawDataDeserializeReader`](https://github.com/asyml/forte/blob/master/forte/data/readers/deserialize_reader.py#L90) as a reader of the pipeline since we expect the input request will be a sequence of serialized `DataPack` strings. This reader is able to deserialize these strings to `DataPack`s.\n",
```

Comment:
Additional ` is rendered, check here: https://asyml-forte--724.org.readthedocs.build/en/724/notebook_tutorial/tutorial_eliza.html#Start-a-Pipeline-Service. This happens a few more times below.

Reply:
The links are also not working